src/components/settings/PushNotifications/styles.module.css

```diff
@@ -27,3 +27,7 @@
   background-color: var(--color-secondary-background);
   padding: var(--space-2);
 }
+
+[data-theme='dark'] .globalInfo {
+  background-color: var(--color-info-background);
+}
```

Comment:
I think it would make more sense to use an `<Alert severity="info">` component otherwise we have a blueish background with a green border in dark mode which we don't have anywhere like this.

Reply:
Actually, the background color change was mentioned to `#19252C` in the issue

Reply:
Let's keep it as per the issue then, we can later change it to `info` if needed.